app/src/main/java/name/boyle/chris/sgtpuzzles/GameView.java

```diff
@@ -714,7 +714,7 @@ protected void onSizeChanged(int viewW, int viewH, int oldW, int oldH)
 		}
 	}
 
-	void rebuildBitmap() {
+	void rebuildBitmap(boolean resetZoomMatrix) {
```

Comment:
The one thing I'm worried about is this making the code more spaghetti-ish. But I'm too scared to refactor